date.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
```

Comment:
[OPTIONAL]

- [ ] You've disabled eslint in all your javascript modules. Please, try to fix the eslint, they are meant to enforce best practice rules. In so doing, you'll learn valuable lessons.